src/testing/jest/jest-27-and-under/jest-setup-test-framework.ts

```diff
@@ -93,7 +93,7 @@ export function jestSetupTestFramework() {
  *
  * @param node The mocked DOM node that will be removed from the DOM
  */
-export function removeDomNodes(node: MockNode) {
+export function removeDomNodes(node: MockNode | undefined) {
```

Comment:
should this be `| undefined | null` ? just wondering based on the `== null` below, which I know will coerce `undefined` correctly, but I _think_ if it's typed like this there would still be a SNC error at the call site if you did `removeDomNodes(null)` and the `== null` sort of implies that could happen?

Reply:
It probably should be - at the moment, it's not contributing to any SNC's (on this branch) at the moment, which is interesting 🤔 I'll update it!

Reply:
cf2c060ea